microsetta_interface/implementation.py

```diff
@@ -2782,8 +2782,15 @@ def get_interactive_account_search(email_query):
     if do_return:
         return email_diagnostics
 
-    accounts = [{"email": acct['email'], "account_id": acct['id']}
-                for acct in email_diagnostics['accounts']]
+    accounts = [{
+        "email": acct['email'],
+        "account_id": acct['id'],
+        "authrocket_status": (
+            "Missing"
+            if acct['auth_issuer'] is None and acct['auth_sub'] is None
+            else "Authenticated"
+        )
+    } for acct in email_diagnostics['accounts']]
```

Comment:
Could this be written as a regular loop without the ternary? Use of a comprehension here, with the ternary, is difficult to read

Reply:
I dont think the implication is authenticated, but that we have iss and sub from authrocket, which I _think_ only occurs once the user has verified their email but it would be helpful to double check

Reply:
> I dont think the implication is authenticated, but that we have iss and sub from authrocket, which I _think_ only occurs once the user has verified their email but it would be helpful to double check

@cassidysymons

Reply:
@wasade simplified comprehension in a new commit; @cassidysymons could you help double check the authrocket logic, thanks!!!